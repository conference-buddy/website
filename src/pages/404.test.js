import React from "react";
import NotFoundPage from "./404";
import { render, cleanup } from "@testing-library/react";

describe("<NotFoundPage />", () => {
  afterEach(cleanup);

  it("shows all elements", () => {
    const component = render(<NotFoundPage />);
    expect(component.asFragment()).toMatchSnapshot();
  });
});
