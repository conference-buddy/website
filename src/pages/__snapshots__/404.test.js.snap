// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<NotFoundPage /> shows all elements 1`] = `
<DocumentFragment>
  <div
    class="p-content-section"
  >
    <h1>
      NOT FOUND
    </h1>
    <p>
      You just hit a route that doesn't exist... the sadness.
    </p>
  </div>
</DocumentFragment>
`;
